services/app-api/utils/formTemplates/formTemplates.ts

```diff
@@ -22,7 +22,7 @@ import {
   ReportType,
 } from "../types";
 // utils
-import { getTemplate } from "../../handlers/formTemplates/populateTemplatesTable";
+// import { getTemplate } from "../../handlers/formTemplates/populateTemplatesTable";
```

Comment:
I commented out a few things to ensure copyover always pulled a new report template so we can change the existing one in place and test changes